### ISSUE TITLE
Set video size to 100%

### ISSUE THREAD

**Issue by mcgowanb** [AckerApple/ack-angular-webcam#13]
The options set only allow for the image to be set in px as a numeric value

>  this.options = {
>       audio: false,
>       video: true,
>       // fallback: true,//force flash
>       width: 420,
>       height: 340,
>       //fallbackQuality: 200,
>       fallbackSrc: 'fallback/jscam_canvas_only.swf'
>     };

I'm using bootstrap and want to set the width to 100% and the height to auto. How can I do this?

**Comment by AckerApple** [comment]
Answer: Don't set the height or width

The width and height will be calculated to grow 100% of the offset nativeElement.

See the code here:
https://github.com/AckerApple/ack-angular-webcam/blob/master/lib/webcam.component.ts#L163

**Comment by mcgowanb** [comment]
I must be missing something, the parent div is set to a width (dynamically) 585x500 and the ack-webcam component width and height are 320x240 ( the default) with no dimensions set in the options array

**Comment by AckerApple** [comment]
Yes, you are indeed missing something and that would be assuming how the code would be processed.

The 320x240 is a failure fallback incase for some reason the nativeElement offset width height is undefined (which should never happen...well unless the height comes back as zero which happens depending on HTML layout and parent height specification in-which case height does become 240).

Try it out. Let me know........

One more thing that you maybe missing. In HTML5, a custom element is like less than an inline element. SO, you may need to do something like:
```
<ack-webcam style="display:block;width:100%:height:100%"></ack-webcam>
``` 

... Please also know that certain conditions exists in-which height 100% results in a zero height (in-which case the 240 kicks in)

**Comment by mcgowanb** [comment]
It doesn't work. Must be to do with the configuration that I have in my html.
I've solved it in the short term by manually modifying the width when ngViewInit is called and grabbing the width of the parent div at that point

**Comment by AckerApple** [comment]
It's working the way I'm telling you on the demo page, see here:
https://ackerapple.github.io/ack-angular-webcam/

As you resize the window, as well, the element resizes accordingly

Here is the code that drives that demo, with the width height 100% spec:
https://github.com/AckerApple/ack-angular-webcam/blob/master/example/app/app.component.ts#L10

**Comment by AckerApple** [comment]
If you wanted to do a pull request for an bind attribute of something like "useParentWidthHeight", I think that would be a good idea.

I'll add it as a feature request issue ( #14 )

**Comment by AckerApple** [comment]
**useParentWidthHeight** is now available in version 1.5.0
